indico/core/db/sqlalchemy/protection.py

```diff
@@ -365,7 +365,7 @@ def can_manage(self, user, permission=None, allow_admin=True, check_parent=True,
         if permission is not None and permission != 'ANY' and permission not in get_available_permissions(type(self)):
             raise ValueError("permission '{}' is not valid for '{}' objects".format(permission, type(self).__name__))
 
-        if user is None:
+        if user is None or user.is_system:
```

Comment:
Maybe a separate `if` with a comment explaining that the system user has no email and thus some access checks (against groups) may fail, and the system user never needs special access, would be better?

Reply:
Sounds better, yes